runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -269,6 +264,22 @@ else if (!config.scoutOverlayAtBank())
 		}
 	}
 
+	private void postRaidsLayout()
```

Comment:
This needs to be updated, after chat manager changes this will no longer work and should be removed (the caching of colors part)

Reply:
Caching of colors has been removed.